### DISCUSSION TITLE
Active Directory auth does not work

### DISCUSSION THREAD

**Discussion by fsainovich** [requarks/wiki#6158] [Error / Bug Report]
I tried to enable Actve Directory auth and it does not work.

After enabled debug ldap flag I get the follow WARN:

wiki  | 2023-02-13T15:21:52.334Z [MASTER] warn: invalid attribute name

I used the following attributes for Field Mapping:

 - sAMAccountName
 - userPrincipalName
 - displayName
 - thumbnailPhoto

ALL fields exists in Active Directory and no more information have shown in the log output.

Thank you in advance

**Comment by D4mocl35** [comment]
I have the same problem, I can't find any solution to my problem among the meager solutions proposed here and there on this forum

Thank you in advance

**Reply by D4mocl35** [reply]
I can't say how I solved my problem because I tried so many different things but it works for me.

The problem is really the detail of the logs which is not up to the level needed to be able to properly debug.

The only advice I can give you is to start from scratch by testing step by step.

Another concept to take into account is that if you do not activate self-registration, you will not be able to connect. Because although LDAP authentication works, WIKIJS must be able to give you access to an account among its accounts in its database. If no account is entered and the function to automatically create one is not activated, then it will not work.
